core/packages/ads/shared/components/Ad/DoubleClick/index.js

```diff
@@ -2,8 +2,11 @@ import React, { PureComponent, Fragment } from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
 import { Helmet } from 'react-helmet';
+import Script from '../../Script';
+
+// eslint-disable-next-line
+import call from 'raw-loader!babel-loader?forceEnv=devClient!./functions/call';
```

Comment:
This loader is also obsolete. https://github.com/frontity/frontity/pull/118/files/2839af4f9caeaa6936375cea3ed816d1e4b1864c#r225954813